src/main/java/uk/gov/hmcts/cmc/claimstore/services/CountyCourtJudgmentService.java

```diff
@@ -69,6 +71,10 @@ public Claim save(final long submitterId, final CountyCourtJudgment countyCourtJ
             );
         }
 
+        if (isCountyCourtJudgmentInputValid(countyCourtJudgment)) {
```

Comment:
can this be done by a validator instead?

Reply:
Is there any generic way of cross field validation like this one or we have to build it ourselves like that: http://dolszewski.com/java/cross-field-validation/

Reply:
Try a class level validator but don't do the reflection approach.

We haven't been doing this sort of validation much because it get's quite complicated and T2 has slightly different business rules to us so it makes it even more difficult in applying this consistently.